turbopack/crates/turbo-tasks-backend/src/backend/mod.rs

```diff
@@ -201,33 +201,43 @@ impl TurboTasksBackendInner {
         unsafe { ExecuteContext::new_with_tx(self, tx, turbo_tasks) }
     }
 
+    #[inline]
     fn suspending_requested(&self) -> bool {
         (self.in_progress_operations.load(Ordering::Relaxed) & SNAPSHOT_REQUESTED_BIT) != 0
     }
 
+    #[inline]
```

Comment:
Unless you have benchmarks one way or the other (it sounds like you don't?), I think you should leave off the `#[inline]` annotation. LLVM should be pretty good about determining when it should inline or not, and these are probably already inlined because they're small functions. All `#[inline]` does is tweak LLVM's thresholds.

```suggestion
    fn suspending_requested(&self) -> bool {
        (self.in_progress_operations.load(Ordering::Relaxed) & SNAPSHOT_REQUESTED_BIT) != 0
    }

```